src/nidmm/examples/nidmm_measurement_grpc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import nidmm
+import sys
+
+
+def example(resource_name, option_string, function, range, digits, address, port):
+    session_name = ''  # user-specified name; empty string means use a new, unnamed session
+
+    # Connect to the grpc server
```

Comment:
Be precise in the terminology. What is "grpc server" ?
Can you spell out the correct product name?

Paraphrasing leads to ambiguity and confusion.

Reply:
Updated with more explicit comment